packages/bootstrap-4/test/Form.test.tsx

```diff
@@ -198,3 +198,35 @@ describe("single fields", () => {
     expect(tree).toMatchSnapshot();
   });
 });
+
+describe("refs", () => {
```

Comment:
Is the test here essentially doing the same thing as the tests in the material-ui / fluent-ui tests? I wonder if they're redundant and we can just create the single test in core perhaps (where we use `withTheme` to create a simple theme). This would simplify maintenance as we wouldn't have to add this test to every new theme that uses TypeScript.

Reply:
yep - i'll drop this now that we have the type tests here